karrot/applications/templates/application_accepted.mjml

```diff
@@ -9,6 +9,7 @@
             <mj-column>
                 <mj-text>
                     <h1>{% trans group_name=group.name %}Your application to join {{ group_name }} was accepted{% endtrans %}</h1>
+                    <p> {% trans group_msg=group.welcome_message %} {{ group_msg }} {% endtrans %}</p>
```

Comment:
This can't be translated, you can remove the `trans` tags and directly write `group.welcome_message` into the double curly braces.

Reply:
That makes perfect sense of course since this is a user input. Translating would require some trickery with the Google Translate API perhaps. :-) But I assume that the admin(s) will write the welcome message in the local language.

I have made the change now.